src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1211,115 @@
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
 
+# region connected-registry
+helps['acr connected-registry'] = """
+type: group
+short-summary: Manage Edge/On-Prem Container Registries with Azure Container Registries.
+"""
+
+helps['acr connected-registry create'] = """
+type: command
+short-summary: Create a connected registry for an Azure Container Registry.
+examples:
+  - name: Create a connected registry in registry mode with access to repos app/hello-world and service/mycomponent. It'll create a sync token and scope-map with the right repo permissions.
+    text: |
+        az acr connected-registry create --registry mycloudregistry --name myconnectedregistry
+            --repository "app/hello-world service/mycomponent"
+  - name: Create a mirror connected registry with only read permissions and pass the sync token
+    text: |
+        az acr connected-registry create --registry mycloudregistry --mode mirror
+            --parent myconnectedregistry --name mymirroracr --sync-token mySyncTokenName
+  - name: Create a mirror connected registry with client tokens and audit logs disabled, that syncs every day at midninght and sync window of 4 hours.
+    text: |
+        az acr connected-registry create --registry mycloudregistry --mode mirror
+            --parent myconnectedregistry --name mymirroracr --repository app/mycomponent
+            --sync-schedule "0 12 * * *" --sync-window PT4H --sync-audit-logs-enabled false
+            --client-tokens myTokenName1 myTokenName2
+"""
+
+helps['acr connected-registry delete'] = """
+type: command
+short-summary: Delete a connected registry from Azure Container Registry.
+examples:
+  - name: Delete a mirror connected registry 'myconnectedregistry' from parent registry 'mycloudregistry' and skip verification.
+    text: |
+        az acr connected-registry delete --registry mycloudregistry 
+            --name myconnectedregistry --yes
+"""
+
+helps['acr connected-registry install-info'] = """
+type: command
+short-summary: Retrives information required for to activate a connected registry.
+examples:
+  - name: Prints the values requiered to activate a connected registry in json format
+    text: |
+        az acr connected-registry install --registry mycloudregistry 
+            --name myconnectedregistry 
+  - name: Prints the values in json format requiered to activate a connected registry and generates the sync token credentials.
+    text: |
+        az acr connected-registry install --registry mycloudregistry 
+            --name myconnectedregistry --fresh-install
+"""
+
+helps['acr connected-registry deactivate'] = """
+type: command
+short-summary: Deactivate a connected registry from Azure Container Registry.
+examples:
+  - name: Deactivate a connected registry 'myconnectedregistry' and skip verification.
+    text: |
+        az acr connected-registry deactivate --registry mycloudregistry 
+            --name myconnectedregistry --yes
```

Comment:
@rosanch Why do we have a `--yes` flag for this command?

Reply:
I see, it is for confirmation.

Reply:
correct. --yes flag is to skip the delete verification.